src/components/MessageActionsList/MessageActionsList.tsx

```diff
@@ -0,0 +1,69 @@
+/*! Copyright (c) Microsoft Corporation. All rights reserved. Licensed under the MIT license. */
+import '../../yamui';
+import * as React from 'react';
+import { LayoutList, LayoutListItem, GutterSize } from '../LayoutList';
+import { CustomizableComponentProps, defaultTheme, customizable } from '../Customizer';
+import { join } from '../../util/classNames';
+import { MessageActionsListProps } from './MessageActionsList.types';
+import { getClassNames } from './MessageActionsList.styles';
+import MenuButton from '../MenuButton';
+import ActionLink from '../ActionLink';
+import { FixedGridRow, FixedGridColumn } from '../FixedGrid';
+import Block, { TextSize } from '../Block';
+import { IconSize } from '../Icon';
+
+/**
+ * A set of links that dispaly in a horizontal list below a message post.  When a maxVisibleItemCount
+ * prop is provided, items that exceed this value will show in a overflow menu.
+ */
+export class MessageActionsList extends React.Component<MessageActionsListProps & CustomizableComponentProps> {
```

Comment:
it would be nice to name this something more generic so can be associated with a content item rather than Message. i'd expect to see yammer-web-components owning a `<MessageActionsList>` component which feeds the correct message-specific actions to this reusable UI component. maybe this could just be HorizontalActionList, or CompactActionList, ContentActionList... i don't think i'm coming up with any winners though

Reply:
I agree. @cainhimself do you have any ideas or opinions here?